src/dashboard/cd-dashboard-children.vue

```diff
@@ -155,5 +158,25 @@
       padding: 0 0 0 5px;
     }
   }
+
+  @media (max-width: @screen-xs-max) {
+    .cd-dashboard-children {
+      max-width: 100%;
+
+      &__divider {
+        border-color: #CAC4CE;
```

Comment:
Same than other PR, don't we have an existing variable for this color ?

Reply:
Not that I know of, correct me if i am wrong

Reply:
it was the same color used with the coderdojo.com dividers

Reply:
Hum. None of them https://github.com/CoderDojo/cd-common/blob/master/common/_colors.less#L61 fits ?